webrender_traits/src/types.rs

```diff
@@ -191,6 +191,53 @@ pub struct ColorF {
 }
 known_heap_size!(0, ColorF);
 
+#[derive(Clone, Copy, Hash, Eq, Debug, Deserialize, PartialEq, PartialOrd, Ord, Serialize)]
+pub struct ColorU {
```

Comment:
We need it for GlyphKey :(. Otherwise yes, I would've just passed ColorF around until we were inside webrender.